### ISSUE TITLE
Duplicate uv --app message improvement

### ISSUE THREAD

**Issue by stefanofusai** [astral-sh/uv#9883]
I've noticed (OCD...) that when doing `uv --app <project>`, when `<project>` already exists, the error message could be improved.

Currently, this is what is output:
![image](https://github.com/user-attachments/assets/2275b12c-6d41-40bf-89ef-fba70953bbbf)

This could be changed to `error: App is already initialized in ...`

Unfortunately I'm not well versed in Rust so I can't submit a PR, but the relevant code is here: https://github.com/astral-sh/uv/blob/main/crates/uv/src/commands/project/init.rs#L105

Let me know your thoughts!

**Comment by zanieb** [comment]
I actually think this should rename "Project" — it's a general error message. `--app` is just a template for the project. The same error would apply for `--lib`.

**Comment by stefanofusai** [comment]
You're right in that the `Initialized project` message should also be changed, and probably n other references
